---
layout: page
title: Publications
permalink: /Publications/
---

# 2023

1. Kirk, J. S., Wang, J., Tracz, A., Long, M., Rosario, S. R., Ji, Y., Kumar, R., Liu, X., Singh, P. K., Puzanov, I., Chatta, G., Cheng, Q., Huang, J., Wrana, J. L., Lovell, J., Yu, H., Liu, S., Shen, M. M., **Liu, T.**, and Tang, D. G. (2023). Integrated single-cell analysis defines the epigenetic basis of castration-resistant prostate luminal cells. bioRxiv : the preprint server for biology, 2023.03.03.530998. [link](https://doi.org/10.1101/2023.03.03.530998)
2.  Ge, Y., Chen, X., Nan, N., Bard, J., Wu, F., Yergeau, D., **Liu, T.**, Wang, J., and Mu, X. (2023) Key transcription factors influence the epigenetic landscape to regulate retinal cell differentiation.  Nucleic Acids Res. 51 (5), 2151-2176

# 2022

1. Wang, J., Morgan, W., Saini, A., **Liu, T.**, Lough, J., and Han, L. (2022) Single-cell transcriptomic profiling reveals specific maturation signatures in human cardiomyocytes derived from LMNB2-inactivated induced pluripotent stem cells. Front Cell Dev Biol. Nov 28;10:895162. doi: 10.3389/fcell.2022.895162. eCollection 2022.
2. Hagiwara, M., Fushimi, A., Bhattacharya, A., Yamashita, N., Morimoto, Y., Oya, M., Withers, HG., Hu, Q., **Liu, T.**, Liu, S., Wong. KK., Long. MD., and Kufe. D. (2022) MUC1-C integrates type II interferon and chromatin remodeling pathways in immunosuppression of prostate cancer. Oncoimmunology.11(1):2029298.
3. Bhattacharya, A., Fushimi, A., Yamashita, N., Hagiwara, M., Morimoto, Y., Rajabi, H., Long, M., Abdulla, M., Ahmad, R., Street, K., Liu, S., **Liu, T.**, and Kufe, D. (2022) MUC1-C Dictates JUN and BAF-mediated Chromatin Remodeling at Enhancer Signatures in Cancer Stem Cells. Mol Cancer Res. [link](https://doi.org/10.1158/1541-7786.MCR-21-0672)

# 2021

1. Zhang, H., Song, L., Wang, X., Cheng, H., Wang, C., Meyer, C., **Liu, T.**, Tang M., Aluru, S., Yue, F., Liu, XS., and Li, H. (2021) Fast Alignment and Preprocessing of Chromatin Profiles with Chromap. Nat Commu. 12(1):6566.
1. Wu, F., Bard, J.E., Kann, J., Yergeau, D., Sapkota, D., Ge, Y., Hu, Z., Wang, J., **Liu, T.**, and Mu, X. (2021) Single cell transcriptomics reveals lineage trajectory of retinal ganglion cells in wild-type and Atoh7-null retinas. Nat Commun. 12(1):1465.
1. Zhou, Y., Bastian, I.N., Long, M.D., Dow, M., Li, W.,**Liu, T.**, Ngu, R.K., Antonucci, L., Huang, J.Y., Phung, Q.T., Zhao, X.H., Banerjee, S., Lin, X.J., Wang, H., Dang, B., Choi, S., Karin, D., Su, H., Ellisman, M.H., Jamieson, C., Bosenberg, M., Cheng, Z., Haybaeck, J., Kenner, L., Fisch, K.M., Bourgon, R., Hernandez, G., Lill, J.R., Liu, S., Carter, H., Mellman, I., Karin, M., and Shalapour, S. (2021) Activation of NF-κB and p300/CBP potentiates cancer chemoimmunotherapy through induction of MHC-I antigen presentation. Proc Natl Acad Sci U S A. 118(8).
1. Amobi-McCloud, A., Muthuswamy, R., Battaglia, S., Yu, H., **Liu, T.**, Wang, J., Putluri, V., Singh, P., Qian, F., Huang, R.Y., Putluri, N., Takemasa, T., Lugade, A., Liu, S., and Odunsi, K. (2021) IDO1 expression in ovarian cancer induces PD-1 in T cells via aryl hydrocarbon receptor activation. Front Immunol. 12:678999

# 2020

1. Wang, CF., Sun, DQ., Huang, X., Wan, CX., Li, ZY., Meyer, C., **Liu, T.**, and Liu, XS. Integrative Analyses of Single-cell Transcriptome and Regulome using MAESTRO. Under revision at Genome Biology
1. Humphrey, K.M., Zhu, L., Hickman, M.A., Hasan, S., Maria, H.,**Liu, T.**, and Rusche, L.N. Evolution of Distinct Responses to Low NAD+ Stress by Rewiring the Sir2 Deacetylase Network in Yeasts. (2020) Genetics

# 2019

1. Chen, H.M., Sackton, T., Wang, J., Mutlu B., Levine, E., **Liu, T.** and Mango, S.E. A heterochromatic histone methyltransferase lowers nucleosomeoccupancy at euchromatic promoters. Under peer-review at Development.
1. Tarbell, E. and **Liu, T.** (2019) HMMRATAC, The Hidden Markov ModeleR for ATAC-seq. Nucleic Acid Research 47(16), e91}
1. O’Brown, Z.K., Boulias, K., Wang, J., Wang, S.Y., O’Brown, N.M., Hao, Z., Shibuya, H., Fady, P.-E., Shi, Y., He, C., Megason, S.G., **Liu, T.**, and Greer, E.L. (2019). Sources of artifact in measurements of 6mA and 4mC abundance in eukaryotic genomic DNA. BMC Genomics 20(1), 445
1. Wang, Z.J., Zhong, P., Ma, K., Seo, J.S., Yang, F., Hu, Z., Zhang, F., Lin, L., Wang, J., **Liu, T.**, Mata, E., Greengard, P., and Yan, Z. (2019). Amelioration of autism-like social deficits by targeting histone methyltransferases EHMT1/2 in Shank3-deficient mice. Mol Psychiatry doi:10.1038/s41380-019-0351-21. Daivs, RB., Likhite, N., Jackson, CA., **Liu, T.**, Yu, MC. (2019) Robust repression of tRNA gene transcription during stress requires protein arginine methylation. Life Sci Alliance. 3;2(3). pii: e201800261.

# Prior to 2019

1. Zhang, P., He, D., Xu, Y., Hou, J., Pan, B.F., Wang, Y., **Liu, T.**, Davis, C.M., Ehli, E.A., Tan, L., Zhou, F., Hu, J., Yu, Y., Chen, X., Nguyen, T.M., Rosen, J.M., Hawke, D.H., Ji, Z., Chen, Y. (2017). Genome-wide identification and differential analysis of translational initiation. Nature Communications 8(1), 1749. 
1. Zhu, L., Jiang, K., Webber, K., Wong, L., **Liu, T.**, Chen, Y., Jarvis, J. (2017). Chromatin landscapes and genetic risk for juvenile idiopathic arthritis. Arthritis Research \& Therapy 19(1), 57. (first author Dr. Zhu is Liu’s postdoc)
1. Mei, S., Qin, Q., Wu, Q., Sun, H., Zheng, R., Zang, C., Zhu, M., Wu, J., Shi, X., Taing, L., **Liu, T.**, Brown, M., Meyer, C.A., Liu, X.S. (2017). Cistrome Data Browser: a data portal for ChIP-Seq and chromatin accessibility data in human and mouse. Nucleic Acids Research 4;45(D1):D658-D662.
1. Hui-Yuen, J.S., Zhu, L., Wong, L.P., Jiang, K., Chen, Y., **Liu, T.**, Jarvis, J.N. (2016). Chromatin landscapes and genetic risk in systemic lupus. Arthritis Research \& Therapy 18:281. (co-first author Dr. Zhu is Liu’s postdoc)
1. Qin, Q., Mei, S., Wu, Q., Sun, H., Li, L., Taing, L., Chen, S., Li, F., **Liu, T.**, Zang, C., Xu, H., Chen, Y., Meyer, C.A., Zhang, Y., Brown, M., Long, H.W., Liu, X.S. (2016). ChiLin: a comprehensive ChIP-seq and DNase-seq quality control and analysis pipeline. BMC bioinformatics 17, 404.
1. Kapoor, S., Zhu, L., Froyd, C., **Liu, T.**, and Rusche, L.N. (2015). Regional centromeres in the yeast Candida lusitaniae lack pericentromeric heterochromatin. Proc Natl Acad Sci USA 112, 12139–12144. (co-first author Dr. Zhu is Liu’s postdoc)
1. Jiang, K., Zhu, L., Buck, M.J., Chen, Y., Carrier, B., **Liu, T.**, and Jarvis, J.N. (2015). Disease-Associated Single-Nucleotide Polymorphisms From Noncoding Regions in Juvenile Idiopathic Arthritis Are Located Within or Adjacent to Functional Genomic Elements of Human Neutrophils and CD4+ T Cells. Arthritis Rheumatol 67, 1966–1977. (Liu is a co-corresponding author, co-first author Dr. Zhu is Liu’s postdoc)
1. Hsu, H.-T., Chen, H.-M., Yang, Z., Wang, J., Lee, N.K., Burger, A., Zaret, K., **Liu, T.**, Levine, E., and Mango, S.E. (2015). Recruitment of RNA polymerase II by the pioneer transcription factor PHA-4. Science 348, 1372–1376.
1. Ho, J.W.K., Jung, Y.L., **Liu, T.**, Alver, B.H., Lee, S., Ikegami, K., Sohn, K.-A., Minoda, A., Tolstorukov, M.Y., Appert, A., et al. (2014). Comparative analysis of metazoan chromatin organization. Nature 512, 449–452. (co-first author)
1. Meijer, D.H., Sun, Y., **Liu, T.**, Kane, M.F., Alberta, J.A., Rowitch, D.H., Stiles, C.D., and Mehta, S. (2014). An amino terminal phosphorylation motif regulates intranuclear compartmentalization of Olig2 in neural progenitor cells. J. Neurosci. 34, 8507–8518.
1. **Liu, T.** (2014). Use Model-Based Analysis of ChIP-Seq (MACS) to Analyze Short Reads Generated by Sequencing Protein-DNA Interactions in Embryonic Stem Cells. Methods Mol. Biol. 1150, 81–95.
1. Catic, A., Suh, C.Y., Hill, C.T., Daheron, L., Henkel, T., Orford, K.W., Dombkowski, D.M., **Liu, T.**, Liu, X.S., and Scadden, D.T. (2013). Genome-wide map of nuclear protein degradation shows NCoR1 turnover as a key to mitochondrial gene regulation. Cell 155, 1380–1395.
1. Bailey, T., Krajewski, P., Ladunga, I., Lefebvre, C., Li, Q., **Liu, T.**, Madrigal, P., Taslim, C., and Zhang, J. (2013). Practical Guidelines for the Comprehensive Analysis of ChIP-seq Data. PLoS Comput Biol 9, e1003326. (co-first author)
1. Sun, H., Qin, B., **Liu, T.**, Wang, Q., Liu, J., Wang, J., Lin, X., Yang, Y., Taing, L., Rao, P.K., et al. (2013). CistromeFinder for ChIP-seq and DNase-seq data reuse. Bioinformatics 29, 1352–1354.
1. Shin, H., **Liu, T.**, Duan, X., Zhang, Y., and Liu, X.S. (2013). Computational methodology for ChIP-seq analysis. Quantitative Biology 1–17.
1. Feng, J., **Liu, T.**, Qin, B., Zhang, Y., and Liu, X.S. (2012). Identifying ChIP-seq enrichment using MACS. Nat Protoc 7, 1728–1740. (co-first author)
1. Landt, S.G., Marinov, G.K., Kundaje, A., Kheradpour, P., Pauli, F., Batzoglou, S., Bernstein, B.E., Bickel, P., Brown, J.B., Cayting, P., et al. (2012). ChIP-seq guidelines and practices of the ENCODE and modENCODE consortia. 22, 1813–1831.
1. Xu, K., Wu, Z.J., Groner, A.C., He, H.H., Cai, C., Lis, R.T., Wu, X., Stack, E.C., Loda, M., **Liu, T.**, et al. (2012). EZH2 oncogenic activity in castration-resistant prostate cancer cells is Polycomb-independent. Science 338, 1465–1469
1. Chen, Y., Negre, N., Li, Q., Mieczkowska, J.O., Slattery, M., **Liu, T.**, Zhang, Y., Kim, T.-K., He, H.H., Zieba, J., et al. (2012). Systematic evaluation of factors influencing ChIP-seq fidelity. Nat Meth 9, 609–614.
1. Qin, B., Zhou, M., Ge, Y., Taing, L., **Liu, T.**, Wang, Q., Wang, S., Chen, J., Shen, L., Duan, X., et al. (2012). CistromeMap: a knowledgebase and web server for ChIP-Seq and DNase-Seq studies in mouse and human. Bioinformatics 28, 1411–1412.
1. Tang, Q., Chen, Y., Meyer, C., Geistlinger, T., Lupien, M., Wang, Q., **Liu, T.**, Zhang, Y., Brown, M., and Liu, X.S. (2011). A comprehensive view of nuclear receptor cancer cistromes. Cancer Res. 71, 6940–6947.
1. **Liu, T.**, Ortiz, J.A., Taing, L., Meyer, C.A., Lee, B., Zhang, Y., Shin, H., Wong, S.S., Ma, J., Lei, Y., et al. (2011). Cistrome: an integrative platform for transcriptional regulation studies. Genome Biol 12, R83.
1. Feng, J., **Liu, T.**, and Zhang, Y. (2011). Using MACS to Identify Peaks from ChIP-Seq Data. Curr Protoc Bioinformatics Chapter 2, Unit2.14.
1. Wang, X., Laurie, J.D., **Liu, T.**, Wentz, J., and Liu, X.S. (2011). Computational dissection of Arabidopsis smRNAome leads to discovery of novel microRNAs and short interfering RNAs associated with transcription start sites. Genomics 97, 235–243.
1. Feng, D., **Liu, T.**, Sun, Z., Bugge, A., Mullican, S.E., Alenghat, T., Liu, X.S., and Lazar, M.A. (2011). A Circadian Rhythm Orchestrated by Histone Deacetylase 3 Controls Hepatic Lipid Metabolism. Science 331, 1315–1319. (co-first author)
1. Chen, Y., Meyer, C.A., **Liu, T.**, Li, W., Liu, J.S., and Liu, X.S. (2011). MM-ChIP enables integrative analysis of cross-platform and between-laboratory ChIP-chip or ChIP-seq data. Genome Biol 12, R11.
1. **Liu, T.**, Rechtsteiner, A., Egelhofer, T.A., Vielle, A., Latorre, I., Cheung, M.-S., Ercan, S., Ikegami, K., Jensen, M., Kolasinska-Zwierz, P., et al. (2011). Broad chromosomal domains of histone modification patterns in C. elegans. Genome Res 21, 227–236.
1. Gerstein, M., Lu, Z.J., Nostrand, E.L.V., Cheng, C., Arshinoff, B.I., **Liu, T.**, and etc. (the worm modENCODE consortium). (2010). Integrative Analysis of Functional Elements in the Caenorhabditis elegans Genome by the modENCODE Project. Science 330(6012): 1775-1787. (co-first author)
1. Shin, H., **Liu, T.**, Manrai, A.K., Liu, X.S. (2009). CEAS: cis-regulatory element annotation system. Bioinformatics 25 (19):2605-6
1. Kolasinska-Zwierz, P., Down, T., Latorre, I., **Liu, T.**, Liu, X.S., Ahringer, J. (2009). Differential chromatin marking of introns and expressed exons by H3K36me3. Nat Genet 41 (3):376-81
1. Deng, L., Zhang, Y., Kang, J., **Liu, T.**, Zhao, H., Gao, Y., Li, C., Pan, H., Tang, X., Wang, D., Niu, T., Yang, H., Zeng, C. (2008). An unusual haplotype structure on human chromosome 8p23 derived from the inversion polymorphism. Hum Mutat 29(10):1209-16.
1. He, S., Liu, C., Skogerbø, G., Zhao, H., Wang, J., **Liu, T.**, Bai, B., Zhao, Y., Chen, R. (2008). NONCODE v2.0: decoding the non-coding. Nucleic Acids Res 36(Database issue):D170-2.
1. Zhang, Y., **Liu, T.**, Meyer, C.A., Eeckhoute, J., Johnson, D.S., Bernstein, B.E., Nussbaum, C., Myers, R.M., Brown, M., Li, W., Liu, X.S..  (2008). Model-based analysis of ChiP-Seq (MACS).  Genome Biology  9:R137. (co-first author)
1. He, S., Su, H., Liu, C., Skogerbø, G., He, H., He, D., Zhu, X., **Liu, T.**, Zhao, Y., Chen, R. (2008). MicroRNA-encoding long non-coding RNAs. BMC Genomics 9:236.
1. He, H., Wang, J., **Liu, T.**, Liu, X.S., Li, T., Wang, Y., Qian, Z., Zheng, H., Zhu, X., Wu, T., Shi, B., Deng, W., Zhou, W., Skogerbø, G., Chen, R. (2007). Mapping the C. elegans noncoding transcriptome with a whole-genome tiling microarray. Genome Res 17(10):1471-7. (co-first author)
1.  The DNA sequence, annotation and analysis of human chromosome 3. Nature 440(7088):1194-8. (list author)
1. He, H., Cai, L., Skogerbø, G., Deng, W., **Liu, T.**, Zhu, X., Wang, Y., Jia, D., Zhang, Z., Tao, Y., Zeng, H., Aftab, M.N., Cui, Y., Liu, G., Chen, R.  (2006). Profiling Caenorhabditis elegans non-coding RNA expression with a combined microarray. Nucleic Acids Res 34(10):2976-83.
