---
layout: page
title: Research
permalink: /Research/
---

# Research Overview

I am developing and applying computational approaches for transcriptional and epigenetics regulation studies. I developed widely used open-source algorithms, including MACS (cited over 19,000 times according to Google Scholar) to analyze ChIP-seq and ATAC-seq data, and an integrative platform for comprehensive analyses on cis-regulatory elements (Cistrome platform). I was a member of the Data Analysis Center and Analysis Working Group of the ENCODE and modENCODE consortium and was involved in deciphering functional elements through analyzing high-throughput profiles of chromatin factors and in comparing chromatin features between model organisms and the human genome. I have actively participated in the development of ChIP-seq guidelines for broad scientific communities. I am the principal investigator of MACS3 project which is supported by Chan-Zuckerberg Initiative. I am currently the tech leader of the [NCI ARTNet](https://www.nciartnet.org/) Coordinating and Data Management Center, the [NCI CAP-IT](https://www.ncicapit.org/) Data and Resource Coordination Center, and the [NCI CIPNET](https://nci-cipnet.org//) Data and Resource Coordination Center. I am a member in the Developmental Therapeutics program under Cancer Center Support Grant (CCSG) for Roswell Park Comprehensive Cancer Center. My team at Roswell Park focuses on building bioinformatics algorithms for single-cell genomics assays to study transcriptional and epigenetic regulatory mechanisms in cancer. 

## Algorithm development on analyzing epigenomics data 

My ongoing research was focused on building methodologies to analyze high-throughput sequencing data targeting at functional regulatory elements. My colleagues and I explored the system noises existing in ChIP-Seq, and applied a dynamic Poisson model to predict transcript factor binding sites or genomic regions with modified histones. The software MACS, which I am still actively maintaining and developing, is well-known of its accuracy and efficiency. It has been widely used by ChIP-Seq community, also was selected as one of the standard bioinformatics tools used by ENCODE, modENCODE, and Roadmap Epigenome consortium. I also extend my research to build methods for other genomics assays such as CUT&Run and ATAC-seq, and recently for single-cell genomic assays.

## Epigenomics database, knowledgebase and infrastructure building

I have participated in building computational infrastructure to ease the data analysis of next-generation sequencing data. I lead a team from Harvard and Eli Lilly, and developed an interactive web-based platform ‘Cistrome Analysis Pipeline’ where users can upload their own ChIP-Seq and RNA-Seq datasets and access a preselected bioinformatics tools for data analysis. I also worked with my colleagues in Harvard to develop downstream analysis tools, databases for all public available human and mouse ChIP-Seq datasets, and searching engines that were further integrated under the Cistrome project.

## Works in consortiums

I have been participating in many NIH funded consortiums such as ENCODE, modENCODE and cancer moonshots, to understand the transcriptional and epigenetic regulation of model organisms and human. My colleagues and I discovered that the methylation on lysine 36 of histone 3 (H3K36me3) links to the exon-intron structure and splicing events of eukaryotes. We also reported broad chromosomal domains with histone modification patterns in C. elegans genome and elucidated the changes of the organization and deployment of the C. elegans genome during development. Recently, I collaborated with our IOTN moonshot consortium colleagues at Dana-Farber and identified the epigenetics role of MUC-1 in cancer stem cells.

## Collaborative works with biologists to understand epigenetic gene regulation

In addition to the contributions described above, with a team of collaborators, I studied the epigenetic regulation in different development stages and tissues of human and mouse. My collaborators at UPenn and I used a mouse system to discover that the epigenetic regulation through histone deacetylase changes over time within a circadian rhythm driven by intrinsic clock of the cells. In a recent collaboration, my UB collaborator and I applied a combination of chromatin immunoprecipitation, RNA expression assays and single-cell experiments to elucidate the development of retinal ganglion cells in mouse.
