---
layout: page
title: Research
permalink: /Research/
---

# Research overview

I develop and apply computational methods for studying transcriptional and epigenetic regulation. My group builds open-source software for epigenomics and single-cell genomics, with a focus on regulatory mechanisms in cancer.

I developed widely used tools including MACS, which has been cited more than 19,000 times (Google Scholar), and helped build the Cistrome platform for integrative analysis of cis-regulatory elements. I previously served in the ENCODE and modENCODE Data Analysis Center and Analysis Working Group, where I contributed to large-scale analysis of chromatin profiles and cross-species comparisons of regulatory features. I also contributed to ChIP-seq best-practice guidelines used by the broader research community.

I am the principal investigator of the MACS3 project, supported by the Chan Zuckerberg Initiative. I currently serve as technical lead for the [NCI ARTNet](https://www.nciartnet.org/) Coordinating and Data Management Center, the [NCI CAP-IT](https://www.ncicapit.org/) Data and Resource Coordination Center, and the [NCI CIPNET](https://nci-cipnet.org/) Data and Resource Coordination Center. At Roswell Park, I am a member of the Developmental Therapeutics Program under the Cancer Center Support Grant (CCSG).

## Algorithm development for epigenomics data

A major part of my work is method development for high-throughput sequencing data that measure functional regulatory elements. My colleagues and I studied technical noise in ChIP-seq and developed dynamic Poisson-model-based approaches to detect transcription factor binding sites and histone-marked regions. MACS remains a core project in my lab, and we continue to improve it for accuracy, robustness, and speed.

Beyond ChIP-seq, we have developed methods for related assays such as CUT&RUN and ATAC-seq, and we are increasingly focused on method development for single-cell genomic assays.

## Epigenomics databases, knowledgebases, and infrastructure

I have worked on computational infrastructure that lowers the barrier for next-generation sequencing analysis. I led a team across Harvard and Eli Lilly to develop the web-based Cistrome Analysis Pipeline, which allows users to upload ChIP-seq and RNA-seq data and run curated downstream analyses through an integrated interface.

I also worked with colleagues at Harvard to build downstream analysis tools, large-scale databases of public human and mouse ChIP-seq datasets, and search systems that were integrated into the broader Cistrome ecosystem.

## Consortium work

I have participated in multiple NIH-funded consortia, including ENCODE, modENCODE, and Cancer Moonshot programs, to study transcriptional and epigenetic regulation in model organisms and human systems.

In this work, my collaborators and I reported that H3K36me3 is linked to exon-intron structure and splicing in eukaryotes. We also characterized broad chromosomal domains of histone modifications in *C. elegans* and examined how genome organization changes during development. More recently, in collaboration with IOTN Moonshot colleagues at Dana-Farber, we identified epigenetic roles of MUC1 in cancer stem-cell biology.

## Collaborative work with biologists on epigenetic regulation

Alongside method and infrastructure work, I collaborate closely with experimental biologists to study epigenetic regulation across tissues and developmental stages.

With collaborators at UPenn, we used mouse models to show that histone deacetylase-mediated epigenetic regulation changes over circadian time in a cell-intrinsic clock framework. In another collaboration, with colleagues at UB, we combined chromatin immunoprecipitation, RNA expression assays, and single-cell experiments to study retinal ganglion cell development in mouse.
